src/server.ts

```diff
@@ -87,6 +89,29 @@ export default async (startEvents: boolean = true) => {
   app.use(
     requestLogger({
       logger,
+      serializers: {
+        // removing cookie from being logged since it contains refresh and access token
+        req: ({ headers, ...req }: { headers: Record<string, string> }) => ({
+          ...req,
+          headers: {},
+        }),
+        res: (res) => {
+          delete res.headers
+          return res
+        },
+      },
+      genReqId: function (req: express.Request, res: express.Response): UUID {
+        const id: UUID = (req.headers['x-request-id'] as UUID) || (req.id as UUID) || randomUUID()
+
+        if (!logger.bindings().req_id) logger.setBindings({ req_id: id })
```

Comment:
this still looks very suspicious. The logger you are modifying the binding of here is a global instance. I still don't this request isolation is working.

Reply:
it has been removed, no longer needed